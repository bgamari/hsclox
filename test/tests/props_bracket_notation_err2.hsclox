class B {}
var b = B();
print b["wut"];
