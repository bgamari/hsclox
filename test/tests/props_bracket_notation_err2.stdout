Undefined property of 'wut'.
[line 3] in script
